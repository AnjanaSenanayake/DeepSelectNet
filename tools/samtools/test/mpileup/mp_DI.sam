@SQ	SN:z	LN:13
@CO	TAG  CTTAGCA  GGT ref
@CO	 AG  CTTAG**TTGGC s1
@CO	 AG  CTTAG**TT    s2
@CO	 AG  CTTAG***T    s4
@CO	 **AACTTAGCA  GG  s3
@CO	 ***ACTTAGCA  GG  s5
@CO	 01AB2345678AB901 qual
s1	0	z	2	0	7M2D2I3M	*	0	0	AGCTTAGTTGGC	0123456AB901
s2	0	z	2	0	7M2D2I	*	0	0	AGCTTAGTT	0123456AB
s4	0	z	2	0	7M2D1P1I	*	0	0	AGCTTAGT	0123456B
s3	0	z	2	0	2D2I9M	*	0	0	AACTTAGCAGG	AB234567890
s5	0	z	2	0	2D1P1I9M	*	0	0	ACTTAGCAGG	B234567890
