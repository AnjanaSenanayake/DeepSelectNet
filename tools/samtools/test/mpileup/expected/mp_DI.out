z	2	A	5	^!.^!.^!.^!*^!*	000AB
z	3	G	5	...*+2AA*+2*A	111AB
z	4	C	5	.....	22222
z	5	T	5	.....	33333
z	6	T	5	.....	44444
z	7	A	5	.....	55555
z	8	G	5	.-2CA.-2CA.-2CA..	66666
z	9	C	5	***..	AAB77
z	10	A	5	*+2TT*+2TT$*+2*T$..	AAB88
z	11	G	3	...	999
z	12	G	3	..$.$	000
z	13	T	1	C$	1
